src/flag_gems/fused/__init__.py

```diff
@@ -47,4 +48,5 @@
     "rwkv_mm_sparsity",
     "dreglu",
     "reglu",
+    "grouped_topk",
```

Comment:
Please find the right place to insert this operator.
We are supposed to ensure that the list is in alphabetic order.